build.sbt

```diff
@@ -3,6 +3,7 @@ import sbtcrossproject.CrossPlugin.autoImport.crossProject
 
 def scala211 = "2.11.12"
 def scala212 = "2.12.8"
+def scala213 = "2.13.0"
```

Comment:
```suggestion
def scala213 = "2.13.1"
```

Reply:
Unpleasant discovery: scalafmt should have _similar_ Scala version with coursier-interface for `assembly` working 😒

Reply:
Can you elaborate? I thought scala-library was shaded with coursier-interface

Reply:
The problem in `reflect.properties` file (https://github.com/scalameta/scalafmt/issues/1514#issuecomment-540172806). coursier-interface jar contains file with content:

```
#Fri Jun 07 10:55:45 UTC 2019
shell.banner=%n     ________ ___   / /  ___%n    / __/ __// _ | / /  / _ |%n  __\\ \\/ /__/ __ |/ /__/ __ |%n /____/\\___/_/ |_/____/_/ | |%n                          |/  %s
copyright.string=Copyright 2002-2019, LAMP/EPFL and Lightbend, Inc.
version.number=2.13.0
osgi.version.number=2.13.0.v20190604-151517-VFINAL-43e040f
maven.version.number=2.13.0
```

But I hope that we can add sbt-assembly rule to discard scala-reflect from coursier-interface.

Reply:
It should be fine to discard the reflect.properties file from coursier-interface, unless the code from that scala-library somehow relies on APIs like `scala.util.Properties.versionNumberString`.